runtime/autoload/man.vim

```diff
@@ -58,6 +58,7 @@ function! man#open_page(count, mods, ...) abort
     else
       execute 'silent keepalt' a:mods 'stag' l:target
     endif
+    setlocal buftype=nofile
```

Comment:
`man#read_page()` (called by `autocmd BufReadCmd man://*`) should already be doing this.

But BufReadCmd doesn't fire for already-existing `man://` buffers. 
```suggestion
    call s:set_options()
```